Include passphrase and 2fa rotation instructions in Admin guide

rocodes opened the pull request:
## Status

Work in progress

## Description of Changes

Fixes #3387. 
Based off of and replaces #3638.  

Changes proposed in this pull request: Add passphrase and 2fa rotation instructions to Admin guide (thanks to @b-meson for writing these!) 

## Testing

Visual review - docs only  
## Deployment
n/a

## Checklist

### If you made changes to documentation:

- [x] Doc linting (`make docs-lint`) passed locally

rocodes commented:
hi @b-meson! It's been a minute :) 
I've cherry-picked your work from #3638 into here. Let me know if you're okay with this, and I'll convert it to Ready for Review. 
Thanks again for doing this writeup.

b-meson commented:
Hi @rocodes thank you for pushing this across the finish line. I think the updated docs look great! I really appreciate your work on this one.

rocodes commented:
> looks good and my comments from #3638 are addressed - I think `docs/images/manual/screenshots/password_reset.png` is unused though, once that is removed this is good to go

removed (and squashed) in 4f0f77e
